### ISSUE TITLE
Delete-Formular

### ISSUE THREAD

**Issue by ejcsid** [xdoo/lhm_animad_admin_html5#215]
Im Rahmen von Ticket #205 wurde diskutiert, ob es wie für CREATE und UPDATE auch ein DELETE Formular geben muss.

Verhalten z.B.:
1. Liste ohne Lösch-Funktionalität (Attribut `hide-delete`)
1. Neuer Lösch-Button, der aus Liste in DELETE-Formular wechselt
1. DELETE-Formular mit Buttons `Löschen` und `Abbrechen`
1. Keine automatische Änderung der Datensätze in der Liste oder Liste leer

**Comment by Baumfrosch** [comment]
Formular heißt hier modalen Fenster mit Abfrage "Willst du den Keeper Hans wirklich unwiderruflich löschen?" --> Löschen | Abbrechen

Oder?

**Comment by xdoo** [comment]
Nein. Formular heißt hier so viel wie Formular mit einer Löschen Schaltfläche :) Der Kontext steht in der Diskussion im Ticket #205.
